pkg/webhook/v1beta1/experiment/validator/validator_test.go

```diff
@@ -256,12 +277,14 @@ func TestValidateExperiment(t *testing.T) {
 	}
 
 	for _, tc := range tcs {
-		err := g.ValidateExperiment(tc.Instance, tc.oldInstance)
-		if !tc.Err && err != nil {
-			t.Errorf("Case: %v failed. Expected nil, got %v", tc.testDescription, err)
-		} else if tc.Err && err == nil {
-			t.Errorf("Case: %v failed. Expected err, got nil", tc.testDescription)
-		}
+		t.Run(tc.testDescription, func(t *testing.T) {
```

Comment:
What advantages we will get by running `t.Run` ?
I think we don't use it in our tests currently.

Reply:
It makes sense. I would like to remove `t.Run`.

Reply:
Done.